client/react/frontpage/components/BlogPage.js

```diff
@@ -169,31 +180,48 @@ const BlogPage = React.createClass({
         }
     }
   },
-  tagToType(tag) {
+  tagToCategory(tag) {
+    tag = tag.toLowerCase().replace(/ /g, '');
     switch (tag) {
-      case 'ConcertReview':
-      case 'ShowReview':
-        return 'CONCERT REVIEW';
-      case 'ArtistInterview':
-        return 'ARTIST INTERVIEW';
-      case 'Sports':
-        return 'SPORTS';
-      case 'FestivalReview':
-        return 'FESTIVAL REVIEW';
-      case 'Other':
-        return 'UCLA RADIO';
+      case 'concertreview':
```

Comment:
Are users the ones who tag these posts on the heroku entry tool? If so, we should be careful in the future about standardizing these tags.

Reply:
In the Heroku entry tool, users will be able to select from categories, they can't directly input the category.

Reply:
I have buncha cases for all the tags mia generated from the tumblr posts